JASP-Engine/JASP/R/commonbayesianttest.R

```diff
@@ -1376,7 +1389,15 @@
     k <- idData + 1
 
 
-    while ((i <= length(x) | j <= length(y)) & k <= length(BF10)) {
+    nTicks <- length(BF10) - i
+    if (plotDifferentPriors)
+      nTicks <- 3L * nTicks
+    startProgressbar(nTicks, sprintf(ngettext(totalPlots,
+                                              "Sequential analysis",
```

Comment:
Maybe "Performing sequential analysis" or something?

Reply:
`nchar("Performing sequential analysis 10 / 11")` has 38 characters. I can see this introducing bugs in specific translations. For example, `nchar("uitvoeren van sequentiële analyse 10 / 11") = 41`, and 41 > 40 so the progress bar will throw an error, but only for a Dutch translation.

Preferably, I'd stay way below the limit of 40 so that translators can freely make longer translations without having to worry about things like these. I'm fine with "Performing sequential analysis", but then maybe we should increase the limit of 40 characters.

The same goes for the robustness check of course.

Reply:
After https://github.com/jasp-stats/jasp-desktop/pull/3863 you get more characters and never a crash

Reply:
This comment is obsolete btw, https://github.com/jasp-stats/jasp-desktop/pull/3853/commits/e9b4f85950aed6004cd13a477f9996f7946c1578 shows I changed it to your suggestion.